conan/tools/gnu/autotools.py

```diff
@@ -48,16 +49,19 @@ def make(self, target=None, args=None):
         str_args = self._make_args
         str_extra_args = " ".join(args) if args is not None else ""
         jobs = ""
-        if "-j" not in str_args and "nmake" not in make_program.lower():
+        jobs_already_passed = re.search(r"-j(\d+)", join_arguments([str_args, str_extra_args]))
```

Comment:
I am not sure if this should be expected. The moment Conan has a standard, defined mechanism to customize the paralelism, this shouldn't be taken into account:
- The profile ``njobs`` should have priority to what the recipe says. In general, anything defined in the profiles should have priorities
- Why would a recipe hardcoded the number of cores it is using in a custom ``arg`` in the helper?

Reply:
Install target doesn't always support multiprocessing in Makefiles (there are examples in conan-center), so recipe must ensure robust installation regardless of njobs in profile.